lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb

```diff
@@ -303,37 +304,50 @@ def cmd_kerberos_ticket_use(*args)
   end
 
   #
-  # Dump all the wifi profiles/credentials
+  # Dump all the shared wifi profiles/credentials
   #
-  def cmd_wifi_list(*args)
-    results = client.kiwi.wifi_list
+  def cmd_wifi_list_shared(*args)
+    interfaces_dir = '%AllUsersProfile%\Microsoft\Wlansvc\Profiles\Interfaces'
```

Comment:
@OJ as this requires elevated privileges shouldn't there be a check if the current user has the correct level of impersonation?